travis/install_grpc.sh

```diff
@@ -22,13 +22,15 @@ fi
 # Python requires a very recent version of virtualenv.
 if [ -n "$grpc_dist" ]; then
   # Create a virtualenv, which also creates a virualenv-boxed pip.
+  # Update both pip and virtualenv.
```

Comment:
Please change the comment such that it explains why we're doing this and include any relevant pointers e.g. URL to a bug.

I think the reason is that the "setuptools" (which is installed by the protobuf package) needs a more recent pip binary. It looks like this is the relevant bug: https://github.com/pypa/setuptools/issues/945